core/base-rest-responses/src/main/java/datawave/webservice/query/exception/DatawaveErrorCode.java

```diff
@@ -258,7 +258,7 @@ public enum DatawaveErrorCode {
     STOPWATCH_MISSING(404, 27, "Stopwatch missing."),
     VALUE_OPERATOR_MISSING(404, 28, "No operator for value."),
     SERVICE_NOT_CONFIGURED(404, 29, "Service is not configured."),
-    // 412 Precondition Failed
+    // 412 Precondition Failed,
```

Comment:
```suggestion
    // 412 Precondition Failed
```

just noticed it to match the format of the codes above